src/util/README.md

```diff
@@ -130,3 +130,17 @@ digraph G {
   2 -> 3 [label="Command line options, file names"];
 }
 \enddot
+
+\subsection ast-examples-section Examples: how to represent the AST of statements, in C and in java
```

Comment:
AST-related things might be better located in the context of language frontends (this is langapi, for now).